### ISSUE TITLE
ESET Flagged "get_shaders.bat"

### ISSUE THREAD

**Issue by Zulu1971** [aa2g/AA2Unlimited#465]
my antivirus flagged the above file with containing a "threat" called **EXP/YAV.Minerva.ef5262**

Downloaded the latest release to attempt a fix for the game crashing after a random amount of time (20-30 min of play)

will be installing this on my secondary rig just in case to see if it is a false flag

**Comment by geneishouko** [comment]
Hello there! Thank you for reporting of this to us. If you take a look at the file here, you'll notice that it hasn't changed in 6 years, not to mention that it does what it claims it does:
https://github.com/aa2g/AA2Unlimited/blob/master/AAUnlimited/Texts/lib/reshade/get_shaders.bat

Please compare the file from this link to the one you got.

Many antivirus engines are known to be very paranoid and false flag happy.

**Comment by Zulu1971** [comment]
Yup, both are identical and after digging through the initial download on my secondary PC i could not find any sign of this threat. False Flag
